Workiva/over_react
# Get examples working in Dart 2

## Ultimate problem:
The `web` directory component demos were not working in the `builder_generate_public_class` (dart 2) branch.

## How it was fixed:
Updated the HTML template file to utilize `index.dart.js` instead of `index.dart`

## Boy Scoutin'
Cleaned up 300+ lints

## Testing suggestions:
1. Pull down this branch
2. Run `pub run build_runner serve` from the root directory
3. Navigate to http://localhost:8080 in your browser, and verify that the component demos render as expected.

## Potential areas of regression:
None expected


---

> __FYA:__ @corwinsheahan-wf @sebastianmalysa-wf @evanweible-wf
